src/core/a-entity.js

```diff
@@ -44,7 +44,9 @@ var proto = Object.create(ANode.prototype, {
       this.states = [];
       this.components = {};
       this.paused = true;
-      this.object3D = new THREE.Mesh();
+      this.object3D = new THREE.Group();
+      this.object3D.el = this;
+      this.objects3D = {};
```

Comment:
somewhat prefer `object3DMap`, `objects3D` implies an array. Or it implies "`objects`, but 3D"